styleguide/source/_patterns/01-molecules/06-components/22-article-preview/article-preview.twig

```diff
@@ -1,9 +1,29 @@
-<div class="topic_article-preview {{ class }}">
-    {% include 'atoms-title-label' with { 'content': 'Topic Label', 'class': 'topic_article-preview_label' } %}
-    {% include "atoms-landscape-3x2" with { 'class': 'topic_article-preview_image' } %}
-    {% include "molecules-link-black-h2" with { 'content': 'Article One Title', 'class': 'topic_article-preview_title' }%}
-    <div class="topic_article-preview_description">
-      {% include "atoms-paragraph" only %}
-    </div>
-    {% include "molecules-article-metadata" with { 'class': 'topic_article-preview_metadata' } %}
+{% set classes = [
+  topic_article.related ? "grid",
+  topic_article.related and topic_article.image ? "topic_article-preview-related-article",
+  topic_article.video ? "topic_article-preview-video"
+] %}
+
+<div class="topic_article-preview {{ classes|join(' ') }} {{ class }}">
+    {% if topic_article.label %}
+      {% include 'atoms-title-label' with { 'content': 'Topic Label', 'class': 'topic_article-preview_label' } %}
+    {% endif %}
+    {% if topic_article.image %}
+      <div class="topic_article-preview_container-image"> {% include "atoms-landscape-3x2" with { 'class': 'topic_article-preview_image', "src": topic_article.image} %}</div>
+    {% endif %}
+    {% if topic_article.video %}
+      <div class="topic_article-preview_container-title"> {% include "molecules-link-black-h2" with { 'class': 'topic_article-preview_title', 'content': topic_article.title }%}</div>
+      {% include "atoms-video-youtube" with {'class': 'topic_article-preview_video'} %}
+      <div class="topic_article-preview_description">
+        {% include "atoms-paragraph" only %}
+      </div>
+    {% else %}
+      <div class="topic_article-preview_container-title"> {% include "molecules-link-black-h2" with { 'class': 'topic_article-preview_title', 'content': topic_article.title }%}</div>
+      {% if not topic_article.related %}
+        <div class="topic_article-preview_description">
+          {% include "atoms-paragraph" only %}
+        </div>
+        {% include "molecules-article-metadata" with { 'class': 'topic_article-preview_metadata' } %}
```

Comment:
Suggestion to move this include outside of this conditional - this fixes the videos not having metadata and is a truer reflection of the content model